tcms/telemetry/templates/telemetry/testing/breakdown.html

```diff
@@ -7,7 +7,10 @@
 {% block contents %}
 
     <div class="container-fluid container-cards-pf">
-
+        <h1 style="margin-top: 0">
+          <span>{% trans "Testing Breakdown" %}</span>
+          <button type="button" class="btn btn-default" data-toggle="tooltip" data-placement="right" title="{% trans 'As a tester and/or manager I want to see a breakdown of tests so that I have an idea how the existing tests for the product are distributed' %}" style="margin-left: 5px">?</button>
```

Comment:
Drop the h1 and button tags, use
`<span class="fa pficon-help"`

- no inline CSS
- we don't need titles
- move the help icon to the right of the screen
- use descriptions from website